### PR TITLE
sap_vm_preconfigure: implement ansible-lint changes

### PR DESCRIPTION
**Pull request by newkit**
Implemented the ansible-lint suggestions.

There are three open issues in code you have added @sean-freeman, could you please take a look?

```
yaml[indentation]: Wrong indentation: expected 4 but found 2
tasks/detect_platform/main.yml:82

yaml[indentation]: Wrong indentation: expected 4 but found 2
tasks/detect_platform/main.yml:115

yaml[empty-lines]: Too many blank lines (1 > 0)
tasks/detect_platform/main.yml:131
```

### CODE REVIEW

File: roles/sap_vm_preconfigure/tasks/platform/hyp_redhat_rhel_kvm_vm/assert-set-tuned-profile.yml [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
   block:
     - name: Get tuned profile
-      command: tuned-adm active
+      ansible.builtin.command: tuned-adm active
       register: __sap_vm_preconfigure_register_tuned_profile_assert
+      changed_when: __sap_vm_preconfigure_register_tuned_profile_assert.rc != 0
 
     - name: Verify tuned profile
-      assert:
+      iansible.builtin.assert:
```
Comment (sean-freeman) [inline roles/sap_vm_preconfigure/tasks/platform/hyp_redhat_rhel_kvm_vm/assert-set-tuned-profile.yml:10]: Typo

**Review by sean-freeman — CHANGES_REQUESTED** [review]
After typo fix, lgtm 👍

**Comment by berndfinger** [comment]
@newkit Please remove the changes for file `roles/sap_vm_preconfigure/.ansible-lint` from this PR in a new commit. The file has already been updated in the meantime with additional changes in the already merged PR https://github.com/sap-linuxlab/community.sap_install/pull/594.

**Review by berndfinger — COMMENTED** [review]
LGTM

**Review by berndfinger — APPROVED** [review]
LGTM

**Review by sean-freeman — APPROVED** [review]
lgtm 👍

**Pull request merged**